.travis.yml

```diff
@@ -12,6 +13,8 @@ matrix:
   allow_failures:
     - rvm: ruby-head
   exclude:
+    - rvm: 1.9.3
+      gemfile: gemfiles/rails_7_devise_4.gemfile
```

Comment:
Travis is not active for this repository anymore, so I wonder if it's worth modifying this file instead of switching to GitHub Actions along the way. I believe only `ruby-head` would work with Rails 7 (if this was being run on Travis), since Rails 7 requires Ruby 2.7+ (see [release notes](https://edgeguides.rubyonrails.org/7_0_release_notes.html)).

Reply:
Yes! Moving to GitHub Actions seems like a good way forward.  I haven't looked much into it but that would be am awesome contribution!

Reply:
The move to GitHub action is done. I added a Ruby 2.7 entry because it is the latest Ruby 2.x.

Reply:
When rebasing this branch onto the latest `master`, this file will cause a conflict, because it does no longer exist in `master`. (The GitHub Actions workflow is configured via `.github/workflows/test.yml`.)

A great way to solve the conflict would be to move your additions to that new file. (You can see what it should look like in [this branch](https://github.com/gonzalo-bulnes/simple_token_authentication/compare/master...OskarEichler-master).)

_Note that using GitHub's "Rebase" button sometimes results in a merge commit. I'd very much rather not have one. If you're not sure how to avoid that, I'm happy to help._